app/Kernel.php

```diff
@@ -153,6 +154,10 @@ protected function buildContainer()
             }
         }
 
+        $backendExtension = new BackendExtension();
+        $container->registerExtension($backendExtension);
+        $extensions[] = $backendExtension->getAlias();
```

Comment:
Hmm, I don't think this `$extensions array` is used. This also means that the two lines above could be put in one line.